api/tests/test_categories.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+
+from api.tests.factories import CategoryFactory
+
+
+@pytest.mark.django_db
+def test_categories_can_be_listed(client):
+    num_items_to_list = 3
+    CategoryFactory.create_batch(num_items_to_list)
+
+    response = client.get('/api/categories/')
+    assert response.status_code == 200
+    assert response.data['count'] == num_items_to_list
+
+
+@pytest.mark.django_db
+def test_a_single_category_can_be_retrieved_by_id(client):
+    category = CategoryFactory()
+    response = client.get('/api/categories/%d/' % category.id)
+    assert response.status_code == 200
+    assert response.data['id'] == category.id
+
+
+@pytest.mark.django_db
+def test_a_single_category_can_be_retrieved_by_name(client):
```

Comment:
interesting point would be what we have to turn off, for example, prevent `DELETE`